modules/base/rotation/timelinerotation.cpp

```diff
@@ -42,11 +41,6 @@ documentation::Documentation TimelineRotation::Documentation() {
         "Timeline Rotation",
         "base_transform_rotation_keyframe",
         {
-            {
-                KeyType,
-                new StringEqualVerifier("TimelineRotation"),
-                Optional::No
-            },
```

Comment:
Just checking to see if these deletions were supposed to be replaced with something else.

Reply:
No, it was due to a discussion I had with @WeirdRubberDuck  and we agreed that it's stupid that we actually check for the Type... since the only way that we would actually get to checking this type is if the type was already correct.  So these checks were always going to be correct anyway and just uncessary overhead